gallery/src/components/NxAccordion/NxAccordionPage.tsx

```diff
@@ -4,184 +4,265 @@
+  NxP,
+  NxH3,
+  NxTile,
+  NxTabs,
+  NxTabList,
+  NxTab,
+  NxTabPanel,
+  NxH2,
+  NxList
+} from '@sonatype/react-shared-components';
 
 import { GalleryDescriptionTile, GalleryExampleTile } from '../../gallery-components/GalleryTiles';
 
 import NxAccordionSimpleExample from './NxAccordionExample';
 import NxAccordionComplexExample from './NxAccordionComplexExample';
 import NxAccordionTertiaryButtonExample from './NxAccordionTertiaryButtonExample';
+import NxStatefulAccordionExample from '../NxStatefulAccordion/NxStatefulAccordionExample';
 
 const NxAccordionSimpleCode = require('./NxAccordionExample?raw'),
     NxAccordionComplexCode = require('./NxAccordionComplexExample?raw'),
-    NxAccordionTertiaryButtonCode = require('./NxAccordionTertiaryButtonExample?raw');
+    NxAccordionTertiaryButtonCode = require('./NxAccordionTertiaryButtonExample?raw'),
+    NxStatefulAccordionCode = require('../NxStatefulAccordion/NxStatefulAccordionExample?raw');
 
-const NxAccordionPage = () =>
-  <>
-    <GalleryDescriptionTile>
-      <NxP>
-        NxAccordion renders a panel with an always-visible header and a collapsible/expandable body section.
-        This is analogous to the HTML 5 <NxCode>&lt;details&gt;</NxCode> element (which it is
-        implemented on top of). There are three related components: <NxCode>NxAccordion</NxCode> itself,
-        <NxCode>NxAccordion.Header</NxCode> which represents the header content, and
-        {' '}<NxCode>NxAccordion.Title</NxCode> which is a convenience component for the header title. All other
-        children of <NxCode>NxAccordion</NxCode> aside from
-        the <NxCode>Header</NxCode> and the <NxCode>Title</NxCode> are rendered in the collapsible section.
+export default function NxAccordionPage() {
+  const [activeTabId, setActiveTabId] = useState(0);
 
-        Note that this component is stateless – its open state must be tracked externally.
-        See <NxCode>NxStatefulAccordion</NxCode> for a version which tracks its own open state.
-      </NxP>
-      <NxTile.Subsection>
-        <NxTile.SubsectionHeader>
-          <NxH3>NxAccordion</NxH3>
-        </NxTile.SubsectionHeader>
-        <NxTable>
-          <NxTable.Head>
-            <NxTable.Row>
-              <NxTable.Cell>Name</NxTable.Cell>
-              <NxTable.Cell>Type</NxTable.Cell>
-              <NxTable.Cell>Required</NxTable.Cell>
-              <NxTable.Cell>Description</NxTable.Cell>
-            </NxTable.Row>
-          </NxTable.Head>
-          <NxTable.Body>
-            <NxTable.Row>
-              <NxTable.Cell>onToggle</NxTable.Cell>
-              <NxTable.Cell>(() =&gt; void)</NxTable.Cell>
-              <NxTable.Cell>No</NxTable.Cell>
-              <NxTable.Cell>
-                A function which gets called when the accordion collapse/expand state is toggled.
+  return (
+    <>
+      <NxTabs activeTab={activeTabId} onTabSelect={setActiveTabId}>
+        <NxTabList aria-label="Table Component Examples">
```

Comment:
I don't think an `aria-label` is needed here - this section has a header (the page title) right above the tabs

Reply:
> I don't think an `aria-label` is needed here - this section has a header (the page title) right above the tabs

https://github.com/sonatype/sonatype-react-shared-components/pull/607/commits/1da396f0ab254dc6305be33e1a44bfee47fc4613